README.md

```diff
@@ -1,2 +1,4 @@
 # IFEMUN
```

Comment:
Nice step bro. I will suggest you make the README file a little more descriptive.

Reply:
Thanks chief. Will do that. Just rushed through that description.